VERSIONS.md

```diff
@@ -1,5 +1,6 @@
 | Version | iOS version | Android version | Common version | Unity IAP compatibility  |
 |---------|-------------|-----------------|----------------|--------------------------|
+| 4.2.0 | 4.11.0 | 5.6.0 | 4.2.1 | Unity IAP 4.4.0+ |
```

Comment:
Note that we are moving to support only Unity IAP 4.4.0. This might warrant a major version... Lmk if you think this should be a major and I can redo this PR.

Reply:
I don't think it warrants a major version, given that Unity IAP isn't part of our API, and that Unity IAP itself isn't outputting a new major version. Our SDK is still API-compatible with the previous version. 
So there should be no code changes if you upgrade, at most you'd have to update your Unity IAP version to a different non-major release